internal/region/region.go

```diff
@@ -1,91 +1,133 @@
 package region
 
-// Region represents the sauce labs region.
-type Region uint
-
-const (
-	// None is an undefined sauce labs region.
-	None Region = iota
-	// USWest1 is a sauce labs region in western US, aka us-west-1.
-	USWest1
-	// USEast4 is a sauce labs region in eastern US, aka us-east-4.
-	USEast4
-	// EUCentral1 is a sauce labs region in the EU, aka eu-central-1.
-	EUCentral1
-	// Staging is a sauce labs internal pre-production environment.
-	Staging
+import (
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
 )
 
-var meta = []struct {
-	Name             string
-	APIBaseURL       string
-	AppBaseURL       string
-	WebDriverBaseURL string
-}{
-	// None
+func init() {
+	homeDir, _ := os.UserHomeDir()
+	path := filepath.Join(homeDir, ".sauce", "regions.yml")
+	file, err := os.Open(path)
+	if err != nil {
+		return
+	}
+	defer file.Close()
+
+	if err = yaml.NewDecoder(file).Decode(&userRegionMetas); err != nil {
+		return
```

Comment:
Might warrant a log line here about this error case. It's mostly for devs at this point, but you probably want to know how badly you messed up your config file :D